src/stable_browser.ts

```
@@ -2919,13 +2919,19 @@ class StableBrowser {
   }
 
   async beforeStep(world, step) {
-    this.stepName = step.pickleStep.text;
-    this.logger.info("step: " + this.stepName);
     if (this.stepIndex === undefined) {
       this.stepIndex = 0;
     } else {
       this.stepIndex++;
     }
+    if (step && step.pickleStep && step.pickleStep.text) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)